sqltree/parser.py

```diff
@@ -2518,7 +2549,7 @@ def _maybe_consume_keyword_sequence(
             keywords_found.append(token)
             if len(keywords_found) == len(keywords):
                 return KeywordSequence(
-                    [Keyword(token, token.text) for token in keywords_found]
+                    [Keyword(token, token.text.upper()) for token in keywords_found]
```

Comment:
I think we shouldn't fix case here, only in the formatter. Ideally it should be possible to exactly reconstruct the SQL from the output of the parser.